amplify/backend/function/expressServer/src/config/default.json

```diff
@@ -1,5 +1,4 @@
 {
 "jwtSecret": "itsasecret",
-"MONGO_URI": "mongodb://admin:pass@ec2-3-249-145-103.eu-west-1.compute.amazonaws.com:27017"
-}
```

Comment:
Not quite sure why this is showing up as a change, as this new URI is already merged to master. Weird but at least it's not changing